iroh/src/discovery/pkarr.rs

```diff
@@ -150,7 +150,7 @@ impl PkarrPublisher {
         debug!("creating pkarr publisher that publishes to {pkarr_relay}");
         let node_id = secret_key.public();
         let pkarr_client = PkarrRelayClient::new(pkarr_relay);
-        let watchable = Watchable::default();
+        let watchable = Watchable::new(None);
```

Comment:
would be nice to implement `Default for Watchable<T: Default>`

Reply:
It already does, I should use it again, though.